config.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
+        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
```

Comment:
We can have both, with proper ontology design.

On Friday, January 16, 2015, Esmé Cowles <notifications@github.com> wrote:

> In config.rdf
> <https://github.com/fcrepo4/ontology/pull/13#discussion-diff-23094609>:
>
> > +<?xml version="1.0"?>
> > +<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
> > +<rdf:RDF
> > +     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
> > +     xmlns:owl="http://www.w3.org/2002/07/owl#"
> > +     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
> > +     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
> > +
> > +    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
> > +        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
> > +        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
> > +    </owl:Ontology>
> > +
> > +    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
> > +        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
> > +        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
>
> @ajs6f <https://github.com/ajs6f> I mean Fedora-the-project. I know that
> namespaces are cheap, and I take the point that there is a certain tidiness
> of having each concern have its own ontology. But as a counter-point, I
> think developers are better served by having a single place to look for
> properties they can set to trigger repository/indexer/etc. behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/ontology/pull/13/files#r23094609>.
>